docs/api.md

```diff
@@ -82,6 +82,57 @@ public class App {
 }
 ```
 
+### Using the Flux API in Gradle
+
+You can use the Flux API in a `build.gradle` Gradle file too. You must first add it as a dependency to your build file:
+
+```
+buildscript {
+  repositories {
+    mavenCentral()
+  }
+  dependencies {
+    classpath "com.marklogic:flux-api:1.0-SNAPSHOT"
+  }
+}
+```
+
+You can then create a custom task that makes use of the Flux API in a similar fashion to using it in Java code:
+
+```
```

Comment:
Do you think you should mention the "import" statement here?

Reply:
Yup, forgot about that. Intellij adds it automatically, but needs to be mentioned.